xml/hwbp_registers.xml

```diff
@@ -493,4 +521,60 @@
             in Table~\ref{tab:action}.
         </field>
     </register>
+
+    <register name="Trigger Extra (RV32)" short="textra32" address="0x7a3">
+        This register is accessible as \Rtdatathree when \Ftype is 2, 3, 4, or
+        5.
+
+        All functionality in this register is optional. The $|value|$ bits may
+        tie any number of upper bits to 0. The $|select|$ bits may only support
+        0 (ignore).
+
+        <field name="mvalue" bits="31:26" access="R/W" reset="0">
+            Data used together with \Fmselect.
+        </field>
+        <field name="mselect" bits="25" access="WARL" reset="0">
+            0: Ignore \Fmvalue.
+
+            1: This trigger will only match if the low bits of
+            \Rmcontext equal \Fmvalue.
+        </field>
+
+        <field name="0" bits="24:18" access="R" reset="0" />
+        <!-- This is space left open for H mode support. -->
+
+        <field name="svalue" bits="17:2" access="R/W" reset="0">
```

Comment:
why so many more bits for svalue vs mvalue?

Reply:
My thinking was that an OS would run in S-mode, and write a PID to svalue (context id). OSs can have many processes.

Similarly I think of the stuff running in M-mode being more akin to a hypervisor, which has VMs or something like them. You don't usually have that many VMs on a system.

I may be totally off-base here.